fwdctl/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 firewood = { version = "0.0.4", path = "../firewood" }
-clap = { version = "4.4.18", features = ["cargo", "derive"] }
```

Comment:
Version updates should happen in a different PR.

Reply:
moved to https://github.com/ava-labs/firewood/pull/521